src/connectors/jamf_inventory.py

```diff
@@ -75,7 +82,39 @@ async def main(table_name):
 
 def ingest(table_name, options):
     global HEADERS
-    creds = options.get('credentials', '')
-    HEADERS = {'Authorization': f'Basic {creds}', 'Accept': 'application/json'}
+    token = getAccessToken(options=options['credentials'])
+    HEADERS = {'Authorization': f'Bearer {token}', 'Accept': 'application/json'}
     return asyncio.get_event_loop().run_until_complete(main(f'data.{table_name}'))
 
+
+# options is a dict containing cliendId and clientSecret
+def getAccessToken(credentials: str) -> str:
+
+    # credentials stored in format {"client_id":"some-id","client_secret":"some-secret"}
+    client_cred = json.loads(credentials)
+
+    jamf_token_api_path = "https://snowflake.jamfcloud.com/api/oauth/token"
```

Comment:
having this as a separate variable used once makes the code harder to read. inline it into the `requests.post` params